src/main/java/com/minecolonies/colony/buildings/BuildingMiner.java

```diff
@@ -159,32 +163,29 @@ public void serializeToView(ByteBuf buf)
         buf.writeInt(currentLevel);
         buf.writeInt(levels.size());
 
-        for(Level level : levels)
+        for (Level level : levels)
         {
             buf.writeInt(level.getNumberOfNodes());
         }
     }
 
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)